website/core/Footer.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
Are we going to fulfil this license stuff?

Reply:
Good catch! Added a license file